src/pds/registrysweepers/utils/__init__.py

```diff
@@ -0,0 +1,214 @@
+import argparse
+import collections
+import functools
+import json
+import logging
+import urllib.parse
+from argparse import Namespace
+from typing import Callable
+from typing import Dict
+from typing import Iterable
+from typing import List
+from typing import Mapping
+from typing import Optional
+from typing import Union
+
+import requests
+
+HOST = collections.namedtuple("HOST", ["cross_cluster_remotes", "password", "url", "username", "verify"])
+
+log = logging.getLogger("registrysweepers.utils")
```

Comment:
[Because I'm a terrible python dev despite using it professionally for half a decade.](https://jpl.slack.com/archives/C3GJ7UUKS/p1683780152598539)

will fix